test/pymmh3_test.py

```diff
@@ -28,7 +28,8 @@ def test_32bit_basic_string( self ):
         with open( os.path.join( file_dir, 'pg1260.txt' ), 'rb' ) as test_file:
             for l in test_file.readlines():
                 s = solution[l]
-                r = pymmh3.hash( l )
+                # A negative value in python has an infinite number of leading 1's in binary unlike c++ where only MSB is 1 in 2's complement form.
+                r = pymmh3.hash( l ) & 0xFFFFFFFF
```

Comment:
I do think that the solution file should be re-generated to build signed values instead of this masking. I think that is a better test since then it actually test what it does.

I might also add comparison-tests against mmh3 if I get time of, comparing return-types and values.